### DISCUSSION TITLE
Include support for alpine linux in the install script

### DISCUSSION THREAD

**Discussion by doshidhaval** [stoplightio/spectral#1591] [Q&A] [answered]
Sometime in Feb, I had created a PR to support installation on alpine https://github.com/stoplightio/spectral/pull/1512. When can this be included in the release?

**Comment by P0lip** [comment, marked as answer]
Ooops, I forgot to include it. My apologies.
I'll cherry-pick the commit when cutting the new release.
In the meantime, I'll create a build manually and attach it to the release.
The script is still available here https://raw.githack.com/stoplightio/spectral/develop/scripts/install.sh.

---
Done, attached the build to the latest release.
LMK if that's okay for you.
The next releases will obviously include the alpine build.

**Reply by doshidhaval** [reply]
I tried using the script from develop branch but it was missing the alpine package. Thanks for the quick fix its working now with the script in develop branch
